analyze/analyses/garbage.rs

```diff
@@ -71,6 +81,32 @@ impl traits::Emit for Garbage {
             obj.field("size_percent", size_percent)?;
         }
 
+        match self
+            .items
+            .iter()
+            .skip(self.limit)
+            .map(|id| &items[*id])
+            .fold((0, 0), |(size, cnt), item| (size + item.size(), cnt + 1))
+        {
+            (size, cnt) if cnt > 0 => {
```

Comment:
Nitpick: I think this would be a bit more readable as

```rust
let (size, cnt) = self
    .items
    .iter()
    .skip(self.limit)
    .map(|id| &items[*id])
    .fold((0, 0), |(size, cnt), item| (size + item.size(), cnt + 1));
if cnt > 0 {
    ...
}
```